src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -403,9 +436,21 @@ def _create_readme(self) -> str:
         readme_url = self._installation.workspace_link('README')
         return readme_url
 
+    def _create_dashboard_links(self, step_name, dashboard_link):
+        dashboards_per_step = [d for d in self._install_state.dashboards.keys() if d.startswith(step_name)]
+        for dash in dashboards_per_step:
+            if len(dashboard_link) == 0:
+                dashboard_link += "Go to the one of the following dashboards after running the job:\n"
```

Comment:
Does this check needs before for loop?

Reply:
@nfx Isn't this valid comment?

Reply:
@nfx Maybe I didn't explained it correctly. this if statement check will happened for every dash, but it get's only initialized just once. Which means say there are 100 dash in dashboards_per_step, this check will be 100 times. If we can move this check before for loop like
```suggestion
            if dashboards_per_step and len(dashboard_link) == 0:
                dashboard_link += "Go to the one of the following dashboards after running the job:\n"
```
this will be checked just once. Hope this is clear.